postgres/postgres.go

```diff
@@ -56,17 +57,24 @@ func Connect(config *CxnConfig, migrations []Migration, env trails.Environment)
 		return nil, err
 	}
 
+	db, err := gormDB.DB()
+	if err != nil {
+		return nil, err
+	}
+
+	db.SetMaxIdleConns(config.MaxIdleCxns)
```

Comment:
I hate this GORM pattern, but yes, this is mutating the parent `gormDB` because of the pointers at hand.